about.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Lato:100,300,400,700,900" type="text/css" >
+    <script src="https://kit.fontawesome.com/8d0a2aa6d4.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/about.css">
+    <title>About section</title>
+</head>
+<body>
+  <header class="header">
+    <ul id="header-list">
+        <li class="header-item icon"><a href="#"><i class="fa fa-facebook"></i></a></li>
+        <li class="header-item icon"><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+        <li class="header-item"><a href="#">English</a></li>
+        <li class="header-item"><a href="#">My Page</a></li>
+        <li class="header-item"><a href="#">Logout</a></li>
+    </ul>
+</header>
+<div class="head-container">
+  <div class="head-image">
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+  <nav class="nav-container">
+      <div class="nav-logo">
+      <img src="images/LogoMakr-3POqWB.png" class="nav-logo-img" alt="home logo">
+      <a href="index.html" class="nav-head">Global Youth <br> Empowerment</a>
+      </div>
+      
+      <div class="nav-dir">
+          <ul class="nav-menu hide">
+              <li class="nav-item">
+                  <a class="nav_link" href="about.html">About</a>
+              </li>
+              <li class="nav-item">
+                  <a class="nav_link" href="#program">Program</a>
+              </li>
+              <li class="nav-item">
+                  <a class="nav_link" href="#join">Join</a>
+              </li>
+              <li class="nav-item">
+                  <a class="nav_link" href="#sponsors">Sponsors</a>
+              </li>
+              <li class="nav-item">
+                  <a class="nav_link" href="#news">News</a>
+              </li>
+          </ul>
+      </div>
+  </nav>
```

Comment:
in the desktop version of your about page navigation there is no link to go back to the main page/homepage please add this feature for more straightforward navigation throughout your webpage 👍🏾 

![image](https://user-images.githubusercontent.com/60612329/217680134-9adc54d2-860f-4446-b429-f2d4edbf9c67.png)